DiscoveryV2.playground/Pages/Introduction.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,69 @@
+//:# Discovery V2
+//: This playground contains code examples for the Discovery V2 service.
+
+//: Please note that Discovery V2 is only available in the Cloud Pak for Data environment.
+//: To learn more about Cloud Pak for Data, go here.
+
+//: To use the Watson Developer Cloud Swift SDK in your application, specify it in your `Cartfile`:
+//:```
+//:github "watson-developer-cloud/swift-sdk"
+//:```
+
+//:Then run the following command to build the dependencies and frameworks:
+//:```bash
+//:carthage update --platform iOS
+//:```
+
+import DiscoveryV2
+
+let username = WatsonCredentials.DiscoveryV2Username
+let password = WatsonCredentials.DiscoveryV2Username
+let cp4dUrl = WatsonCredentials.DiscoveryV2CP4DUrl
+
+//:## Authentication
+let authenticator = WatsonCloudPakForDataAuthenticator(username: username, password: password, url: url)
+
+let discovery = Discovery(version: "2019-11-30", authenticator: authenticator)
```

Comment:
API ref v2 has 2 version dates, neither is this one (2019-11-22 and 2019-11-29). So I'm not sure what the correct version date is.